### DISCUSSION TITLE
how to change title of micropython ? (MicroPython v1.19.1-269-g94a19f106 on 2022-08-15; ESP32 module with ESP32)

### DISCUSSION THREAD

**Discussion by picpic020960** [micropython/micropython#9276] [General]
Bonjour ,
all in the subject : 
i want to costum the title of my build : 
`MicroPython v1.19.1-269-g94a19f106 on 2022-08-15; ESP32 module with ESP32`
how to ?
thanks for help
Regards

**Comment by Lobo-T** [comment]
It's in mpconfigport.h in the board configuration folder: https://github.com/micropython/micropython/blob/4903e48e340bd9741577f30cacb31605cc70cf20/ports/esp32/boards/GENERIC/mpconfigboard.h#L1

**Comment by jimmo** [comment]
The various parts of this title come from different places. e.g. the "ESP32 module" comes from the name in mpconfigboard.h as @Lobo-T said.

The rest of the message is `MICROPY_BANNER_NAME_AND_VERSION` and `MICROPY_BANNER_MACHINE` which you can also override in mpconfigboard.h.
See https://github.com/micropython/micropython/blob/master/py/mpconfig.h#L1717 for the defaults (but you should override them in mpconfigboard.h).
